examples/installation/build.sh

```diff
@@ -6,7 +6,6 @@ Usage: sh $0 command [argument]
 
 command:
   test-all:                        tests all projects in this repo.
-  test-xcode6:                     tests all Xcode 6 projects in this repo.
   test-xcode7:                     tests all Xcode 7 projects in this repo.
```

Comment:
Sorry, what I actually meant was that this adds support for Xcode 8 but there's no `test-xcode8` command added here.